src/components/todo-input/TodoInput.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <v-text-field
+    v-model="value"
+    class="todo-input"
+    placeholder="what needs to be done?"
+    clearable
+  >
+    <v-icon
+      v-if="!!value"
+      icon
+      class="btn-save"
+      slot="append"
+      @click="onAddTodo"
+    >
+      mdi-content-save
+    </v-icon>
+  </v-text-field>
+</template>
+
+<script>
+  export default {
+    name: 'TodoInput',
+
+    data() {
+      return {
+        value: ''
+      }
+    },
+    methods: {
+      onAddTodo() {
+        this.$emit('add-todo', this.value)
+      }
+    }
+  }
+</script>
+<style lang="less" scoped>
+  .input-model {
+    max-width: 50%;
+    margin-right: auto;
+    margin-left: auto;
+  }
+  .todo-input {
+    width: 100%;
+    padding: 16px;
+    font-family: Roboto;
+    font-style: oblique;
+    font-size: 26px;
+    min-width: 600px;
```

Comment:
I don't think the min-width is necessary, the input should just fill all the available space